docs/tutorials/quickstart.md

```diff
@@ -118,18 +129,18 @@ $ aks-engine deploy --subscription-id 51ac25de-afdg-9201-d923-8d8e8e8e8e8e \
     --set servicePrincipalProfile.clientId="47a62f0b-917c-4def-aa85-9b010455e591" \
     --set servicePrincipalProfile.secret="26054d2b-799b-448e-962a-783d0d6f976b"
 
-INFO[0000] new api model file has been generated during merge: /tmp/mergedApiModel619868596
+INFO[0000] new API model file has been generated during merge: /tmp/mergedApiModel619868596
 WARN[0002] apimodel: missing masterProfile.dnsPrefix will use "contoso-apple"
 INFO[0025] Starting ARM Deployment contoso-apple-1423145182 in resource group contoso-apple. This will take some time...
 INFO[0256] Finished ARM Deployment (contoso-apple-1423145182). Succeeded
 ```
 
-`aks-engine` will output Azure Resource Manager (ARM) templates, SSH keys, and a kubeconfig file in `_output/contoso-apple-59769a59` directory:
+`aks-engine` will output ARM templates, SSH keys, and a kubeconfig (A specification that may be used as input to the `kubectl` command to establish a privileged connection to the Kubernetes apiserver) file in `_output/contoso-apple-59769a59` directory:
```

Comment:
Maybe we could link to a definition of "kubeconfig" somewhere instead? I just thought this long parenthetical was hard to read.